Remove code that looks for default Fink and MacPorts prefixes.

Patch in a placeholder for the applications directory which will be replaced
with the real value by the Portfile.
--- configure.orig	2026-02-05 17:26:28
+++ configure	2026-02-06 16:33:10
@@ -991,22 +991,10 @@
             LDFLAGS="-L$prefix/$libdir"
         else
             LDFLAGS="$LDFLAGS -L$prefix/$libdir"
-        fi
-
-        if test "$is_em" != "yes" ; then
-            if test -d /sw/bin ; then
-                alt_macosx_dir="/sw"
-            elif test -d /opt/local/bin ; then
-                alt_macosx_dir="/opt/local"
-            fi
-            if test "$alt_macosx_dir" != "" ; then
-                CFLAGS_DIR="$CFLAGS_DIR -I$alt_macosx_dir -I$alt_macosx_dir/include"
-                LDFLAGS="$LDFLAGS -L$alt_macosx_dir -L$alt_macosx_dir/lib"
-            fi
         fi
 
         CFLAGS="$CFLAGS_DIR $CFLAGS"
-        Mac_Applications="/Applications"
+        Mac_Applications="@APPLICATIONS_DIR@"
         SHFLAGS="-dynamiclib"
         DYN_LIB_SUFFIX=".dylib"
         extralibs=""
